android/app/src/main/java/io/o2mc/app/App.java

```diff
@@ -22,10 +23,14 @@ public static O2MC getO2mc() {
   }
 
   @Override public void onO2MCDispatchException(O2MCDispatchException e) {
-    Log.e(TAG, "onO2MCDispatchException: Some exception: handling from App.java");
+    Log.e(TAG, "Handling an O2MCDispatchException from App.java: " + e.getMessage());
   }
 
   @Override public void onO2MCDeviceException(O2MCDeviceException e) {
-    Log.e(TAG, "onO2MCDeviceException: Some exception: handling from App.java");
+    Log.e(TAG, "Handling an onO2MCDeviceException from App.java: " + e.getMessage());
+  }
+
+  @Override public void onO2MCEndpointException(O2MCEndpointException e) {
+    Log.e(TAG, "Handling an onO2MCEndpointException from App.java: " + e.getMessage());
```

Comment:
inconsistent log message; use `O2MCEndpointException` instead of `onO2MCEndpointException`.